app/assets/javascripts/join_meetup.js

```diff
@@ -0,0 +1,39 @@
+$(".join-button").on('click', function(event) {
+  event.preventDefault();
+
+  $button = $(this)
+  meetupId = this.id.split("-")[1]
+  attendeeAction = $button.text()
+  $.ajax({
+    url: "/meetups/" + meetupId + "/attendees",
+    method: 'POST',
+    dataType: 'json',
+    data: {meetup_id: meetupId, attendeeAction: attendeeAction}
```

Comment:
Identifier 'meetup_id' is not in camel case.<br>'meetupId' is not defined.<br>'attendeeAction' is not defined.